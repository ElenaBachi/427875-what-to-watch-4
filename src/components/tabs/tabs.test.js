import React from "react";
import renderer from "react-test-renderer";

import Tabs from "./tabs.jsx";

const mock = {
  tabList: {
    FIRST: `First Tab`,
    SECOND: `Second Tab`,
    THIRD: `Third Tab`,
  },
};

it(`Tabs should render correctly`, () => {
  const activeTab = Object.keys(mock.tabList)[0];

  const tree = renderer.create(
      <Tabs
        tabList={mock.tabList}
        onTabChange={() => {}}
        activeTab={activeTab}
      />).toJSON();

  expect(tree).toMatchSnapshot();
});
