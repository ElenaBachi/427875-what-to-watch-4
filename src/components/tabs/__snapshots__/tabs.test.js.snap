// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tabs should render correctly 1`] = `
<nav
  className="movie-nav movie-card__nav"
>
  <ul
    className="movie-nav__list"
  >
    <li
      className="movie-nav__item "
      onClick={[Function]}
    >
      <a
        className="movie-nav__link"
        href="#"
      >
        First Tab
      </a>
    </li>
    <li
      className="movie-nav__item "
      onClick={[Function]}
    >
      <a
        className="movie-nav__link"
        href="#"
      >
        Second Tab
      </a>
    </li>
    <li
      className="movie-nav__item "
      onClick={[Function]}
    >
      <a
        className="movie-nav__link"
        href="#"
      >
        Third Tab
      </a>
    </li>
  </ul>
</nav>
`;
